Updating the operational technologies (#63)
IETF-OPSAWG-WG/draft-opsarea-rfc5706bis: draft-opsarea-rfc5706bis.md

   * Move the "Operational Considerations" Appendix A to a Checklist maintained in GitHub

>    * Add a requirement for an “Operational and Management Considerations” section in Internet-Drafts, along with specific guidance on its content.

Reviewer: ```suggestion
   * Add a requirement for an "Operational and Management Considerations" section in all Internet-Drafts, along with specific guidance on its content.
```

   * Update the operational and manageability-related technologies to reflect over 15 years of advancements

>       * Provide focus and details on NETCONF/YANG standards, deprioritizing MIB Modules.

Reviewer: ```suggestion
      * Provide focus and details on YANG-based standards, deprioritizing MIB Modules.
```

##  Available Management Technologies {#sec-mgmt-tech}

>    The IETF provides a number of standardized management protocols suitable for various operational purposes, as outlined in {{?RFC6632}}. Broadly, these include core network management protocols, purpose-specific management protocols, and network management data models. Examples include:

Reviewer: ```suggestion
   The IETF provides a number of standardized management protocols suitable for various operational purposes, as outlined in {{?RFC6632}}. Broadly, these include core network management protocols, purpose-specific management protocols, and network management data models. A non-exhaustive list of such protocols is provided below:
```
